packages/component-library/draft/Kaizen/GuidanceBlock/GuidanceBlock.spec.tsx

```diff
@@ -20,14 +20,9 @@ describe("GuidanceBlock", () => {
           description:
             "Mussum Ipsum, cacilds vidis litro abertis. Suco de cevadiss, é um leite divinis.",
         }}
-        button={{
```

Comment:
Need to double check there weren't also changes in the old one one that need to be kept

Reply:
All good. There were no changes to this file int he old draft since March 5. (we copied the draft packages 15 days ago).